AzureCalling/Views/ParticipantLabelView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@IBDesignable
+class ParticipantLabelView: UIStackView {
+    var insets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+
```

Comment:
What's the insets for?

Reply:
Removed. Accidentally copied this over from ParticipantLabel